src/core.js

```diff
@@ -284,6 +284,16 @@ function done() {
 		total: config.stats.all,
 		runtime: runtime
 	} );
+
+	// Clear own sessionStorage items if all tests passed
+	if ( config.reorder && defined.sessionStorage && config.stats.bad === 0 ) {
+		for ( i = 0; i < sessionStorage.length; i++ ) {
+			key = sessionStorage.key( i++ );
```

Comment:
I think we can address this in another PR. If the implementation is busted, then we should also investigate getting a failing test.